app/assets/stylesheets/components/_page_preview.scss

```diff
@@ -0,0 +1,6 @@
+.app-c-preview {
```

Comment:
There's no need to nest these as the second selector `app-c-preview__desktop-preview` is already unique. The following will be enough:
```
.app-c-preview__desktop-preview {
  width: 100%;
  height: 1000px;
}
```

If we'll have specific styles for `.app-c-preview` we can define them separately
```
.app-c-preview { ... }
```